db/structure.sql

```diff
@@ -1665,3 +1675,6 @@ INSERT INTO schema_migrations (version) VALUES ('20181013103240');
 
 INSERT INTO schema_migrations (version) VALUES ('20181018123139');
 
+INSERT INTO schema_migrations (version) VALUES ('20181025220728');
```

Comment:
Hi @BGRicker 

Maybe my question will seem silly, but why this line was added here?
Migration with timestamp `20181025220728` doesn't existed in this tree https://github.com/EBWiki/EBWiki/tree/29621b6de008309a7b4acc57d4ad865c680d9714/db/migrate

Reply:
hey @d4mk0, not a silly question at all, this was a good catch. It was left over from a previous version of my migration, it should have been removed so thank you for noticing it!